README.md

```
@@ -165,6 +165,88 @@ databaseManager = await CreateDatabaseManager(dbConfig);
+    // Provide the service name here
+    createAuditProvider('case-management-service'), 
+  ],
+  exports: [
+    'AUDIT_LOGGER', // Export the provider by its token
+  ],
+})
+export class AuditLogModule {}
+```
+Then, import this new `AuditLogModule` into the `imports` array of your root `app.module.ts`.
+
+*How to Use*
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)